src/dbus/common/constants.hpp

```diff
@@ -63,6 +63,8 @@
 #define OTBR_DBUS_GET_PROPERTIES_METHOD "GetProperties"
 #define OTBR_DBUS_LEAVE_NETWORK_METHOD "LeaveNetwork"
 #define OTBR_DBUS_SET_NAT64_ENABLED_METHOD "SetNat64Enabled"
+#define OTBR_DBUS_START_EPHEMERAL_KEY_MODE_METHOD "StartEphemeralKeyMode"
```

Comment:
```suggestion
#define OTBR_DBUS_ACTIVATE_EPHEMERAL_KEY_MODE_METHOD "ActivateEphemeralKeyMode"
```

Can we use the words activate / deactivate to stay consistent with the terms used in the Thread spec?

Reply:
Currently we are following the design spec; @librasungirl for more thoughts. Thanks~

Reply:
Change to activate / deactivate per discussion. Thanks~